lib/node_modules/@stdlib/stats/base/snanmeanpn/src/main.c

```diff
@@ -1,7 +1,7 @@
 /**
 * @license Apache-2.0
 *
-* Copyright (c) 2020 The Stdlib Authors.
+* Copyright (c) 2025 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2020 The Stdlib Authors.
```
Git isn't counting this as a new file like it does in some of the other PRs, no need to update the copyright year here